modules/prometheus-exporter_oracle/conf/03-pdbisdown.yaml

```diff
@@ -0,0 +1,19 @@
+module: oracle
+name: "Pluggable database"
+id: pdbisdown
+
+transformation: true
+aggregation: ".mean(by=['job'])"
+
+signals:
+  signal:
+    metric: oracledb_Pdb_is_up_count
```

Comment:
weird, I don't see this metric here https://github.com/iamseth/oracledb_exporter